extensions/cache/src/main/java/org/radargun/stages/cache/test/UUIDKeysSelector.java

```diff
@@ -0,0 +1,29 @@
+package org.radargun.stages.cache.test;
+
+import java.util.Random;
+import java.util.UUID;
+
+import org.radargun.config.DefinitionElement;
+import org.radargun.config.Namespace;
+import org.radargun.stages.test.TestStage;
+
+public class UUIDKeysSelector implements KeySelector {
+
+   public UUIDKeysSelector() {
+   }
+
+   @Override
+   public long next() {
+      return UUID.randomUUID().getMostSignificantBits() & Long.MAX_VALUE;
```

Comment:
Oh my, folks - don't use `UUID.randomUUID()` in performance tests! This uses secure source of randomness and consumes entropy. Eventually you could run out and get blocked in this call.

Check out https://www.baeldung.com/java-security-egd

Reply:
How about
```java
long next = ThreadLocalRandom.current().nextLong() & Long.MAX_VALUE;
```

Reply:
Yes, please do that.